frontend/src/main/java/de/iav/frontend/controller/RegistrationController.java

```diff
@@ -102,6 +158,48 @@ else if (password.getText().length() < 4) {
             else return email.getText().contains(".");
         }
     }
+
+    private boolean isUpdateDataValid(){
+        if (firstName.getText().isEmpty() || lastName.getText().isEmpty()||
+                email.getText().isEmpty()) {
+            return false;
+        }
+        else{
+            if (!email.getText().contains("@"))
+                return false;
+            else if (email.getText().length() < 4) {
+                return false;
+            }
+            else if (firstName.getText().length() < 4) {
+                return false;
+            }
+            else if (lastName.getText().length() < 4) {
+                return false;
+            }
+            else return email.getText().contains(".");
+        }
+    }
+
+    public void setUserToUpdate(User loggedUser, User user) {
+        this.wayToCreateOrEditUser = WayToCreateOrEditUser.UPDATE;
+        userToUpdate = user;
```

Comment:
Bei der Nutzung von `this.` würde ich dir empfehlen es ganzheitlich zu nutzen, oder auch nicht zu nutzen.
Nur, es sollte durchgängig einheitlich sein, weil es sonst etwas verwirrend ist, ob es sich um ein Klassenattribut handelt oder nicht